plugin/evm/syncervm_client.go

```diff
@@ -257,7 +257,7 @@ func (client *stateSyncerClient) syncBlocks(ctx context.Context, fromHash common
 		}
 		blocks, err := client.client.GetBlocks(ctx, nextHash, nextHeight, parentsPerRequest)
 		if err != nil {
-			log.Warn("could not get blocks from peer", "err", err, "nextHash", nextHash, "remaining", i+1)
+			log.Error("could not get blocks from peer", "err", err, "nextHash", nextHash, "remaining", i+1)
```

Comment:
This means that it has failed to sync, so this should be increased from `Warn` to `Error`

Reply:
Wouldn't it be possible for the peer to send us a malformed blocks and trigger this?

Reply:
The call to `client.client.GetBlocks` includes retries so if we exhaust retries we can log `Error` since we will return err and the sync has failed.